cli/cmd/root.go

```diff
@@ -43,6 +45,12 @@ func Execute() {
 	currentLogLevel := logging.LogLevel
 	logging.LogLevel = logging.QuietLevel
 
+	containsHelp := strings.Contains(strings.Join(os.Args, " "), "help")
```

Comment:
Will this also work when we use `-h` instead of `--help`?

Also, with the logic involved this check would look for any param that contains help, e.g., `keptn create service helper --project=carts` would also be caught.

I think we should check if any entry in `os.Args` has `--help` or `-h` in it.

Reply:
I changed it to check for `-h` or `--help` parameter and actually do it by iterating through all parameters instead of doing a `strings.Contains` check to prevent false detection of parameters only starting with `-h`.
Currently there are no to me known parameters starting with `-h`, but in future there might be some introduced.

Reply:
Thanks, looks good!